specs/_features/eip7594/das-core.md

```diff
@@ -221,6 +224,40 @@ def get_data_column_sidecars(signed_block: SignedBeaconBlock,
     return sidecars
 ```
 
+#### `get_extended_sample_count`
+
+```python
+def get_extended_sample_count(allowed_failures: uint64) -> uint64:
```

Comment:
I found that Hypergeometric distribution makes sense only with selection without replacement. I think we should either:
- Require the nodes to do selection without replacement, or
- Note in the spec that if you use selection without replacement, use Hypergeometric distribution, while, if you use the one with replacement, use Binomial distribution.